examples/agn_example.py

```diff
@@ -117,7 +118,7 @@
 y = spec.plotter.ymax * 0.85
 for i, line in enumerate(spec.measurements.lines.keys()):
     x = spec.measurements.lines[line]['modelpars'][1]
-    spec.plotter.axis.plot([x]*2, [spec.plotter.ymin, spec.plotter.ymax], ls = '--', color = 'k')
+    spec.plotter.axis.plot([x]*2, [spec.plotter.ymin.value, spec.plotter.ymax.value], ls = '--', color = 'k')
```

Comment:
Does matplotlib still fail if you leave out the `.value`?  It shouldn't... I'm concerned.